spec/spec_helper_acceptance.rb

```diff
@@ -1,9 +1,8 @@
 # frozen_string_literal: true
 
-require 'puppet_litmus'
-PuppetLitmus.configure!
+require 'voxpupuli/acceptance/spec_helper_acceptance'
 
-require 'spec_helper_acceptance_local' if File.file?(File.join(File.dirname(__FILE__), 'spec_helper_acceptance_local.rb'))
+configure_beaker(modules: :metadata)
 
 shared_examples 'an idempotent resource' do
```

Comment:
We already have that (https://github.com/voxpupuli/voxpupuli-acceptance/?tab=readme-ov-file#an-idempotent-resource), so does this redefine it?

It would be good to at least standardize on the same naming. This uses `pp` while our common one uses the more descriptive `manifest`.

Reply:
Noticed the warnings ...
May be we could just remove this. I'll give it a try.